scipy/special/tests/test_hyp2f1.py

```diff
@@ -0,0 +1,228 @@
+from __future__ import division, print_function, absolute_import
+
+import itertools
+import numpy as np
+
+from scipy.special._testutils import FuncData, MissingModule, check_version
+from scipy.special import hyp2f1
+
+try:
+    import mpmath
+except ImportError:
+    mpmath = MissingModule('mpmath')
+
+# Fix the seed number
+np.random.seed(1108257)
```

Comment:
I don't think this is the correct place to put this. It should go on the first line of `test_hyp2f1`.

Reply:
@xoviat I moved where the random seed is set